File: queuer/src/main/java/com/rndapp/task_feed/QueuerApplication.java
package com.rndapp.task_feed;

import android.app.Application;

import com.android.volley.RequestQueue;
import com.android.volley.toolbox.Volley;

/**
 * Created by eschrock on 2/4/14.
 */
public class QueuerApplication extends Application{
    private RequestQueue queue;

    public RequestQueue getRequestQueue(){
        if (queue == null) queue = Volley.newRequestQueue(this);
        return queue;
    }
}

File: queuer/src/main/java/com/rndapp/task_feed/QueuerApplication.kt
package com.rndapp.task_feed

import android.app.Activity
import android.app.Application
import com.rndapp.task_feed.api.VolleyManager
import com.rndapp.task_feed.managers.SessionManager

/**
 * Created by eschrock on 2/4/14.
 */
class QueuerApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        VolleyManager.init(this)
        SessionManager.preferences = this.getSharedPreferences(SessionManager.API_KEY_PREFERENCE, Activity.MODE_PRIVATE)
    }
}
